### PR TITLE
Adds ability to provide OpenAIKey via secure string

### PR DESCRIPTION
**Pull request by kborowinski**
This PR adds ability to provide OpenAIKey via secure string using new `Set-OpenAIKey` function.

### CODE REVIEW

**Comment by kborowinski** [comment]
@dfinke  This PR fixes #47

File: Public/ai.ps1 [hunk 1]
```diff
@@ -37,7 +37,7 @@ $($inputPrompt)
 $(($lines | Out-String).Trim())
 
 "@
-        
-        Get-GPT3Completion -prompt $fullPrompt.Trim() -max_tokens $max_tokens -temperature $temperature
+
+        (Get-GPT3Completion -prompt $fullPrompt.Trim() -max_tokens $max_tokens -temperature $temperature).Trim()
```
Comment (kborowinski) [inline Public/ai.ps1:41]: This should be a separate PR but it's small and benign.

**Comment by dfinke** [comment]
@kborowinski Looks really good. Dotted the I's and crossed the T's. Will try it out locally when I can and then merge.

Thanks for the PR!
